docs/02_getting-started/03_smart-contract-development/04_data-persistence.md

```diff
@@ -140,13 +139,13 @@ addressbook(name receiver, name code, datastream<const char*> ds):contract(recei
 ```
 
 ## Step 7: Adding a record to the table
-Previously, the primary key of the multi-index table was defined to enforce that this contract will only store one record per user. To make it all work, some assumptions about the design need to be established.
+Previously, the primary key of the multi-index table was defined to enforce that this contract will only store one record per user. To make it all work, some rules about the design need to be established.
 
 1. The only account authorized to modify the address book is the user.
-2. the **primary_key** of our table is unique, based on username
+2. The **primary_key** of our table is unique, based on username
 3. For usability, the contract should have the ability to both create and modify a table row with a single action.
 
-In EOSIO a chain has unique accounts, so `name` is an ideal candidate as a **primary_key** in this specific use case. The [name](https://developers.eos.io/manuals/eosio.cdt/latest/structeosio_1_1name) type is a `uint64_t`.
+On a EOSIO blockchain an account name is unique, therefore the `name` type is an ideal candidate as a **primary_key**. Behind the scenes, the [name](https://developers.eos.io/manuals/eosio.cdt/latest/structeosio_1_1name) type is an `uint64_t` integer.
 
```

Comment:
I would reformulate this
Behind the scenes, the [name](https://developers.eos.io/manuals/eosio.cdt/latest/structeosio_1_1name) type is an `uint64_t` integer.

to this
Behind the scenes, the [name](https://developers.eos.io/manuals/eosio.cdt/latest/structeosio_1_1name) is of `uint64_t` type.

Reply:
I actually think the first form is easier to understand and more direct.

Reply:
yeah ok, to me, saying "uint64_t integer" is a pleonasm, just saying uint64 you alredy say it is integer type.
no biggy you can leave it like this.

Reply:
well there are multiple of integer types and our audience are not necessary the season C++ developers.

But I was just more concern of " is of " phase to me it is a bit less easy to read

Reply:
if your agument/concern is "there are multiple of integer types", then you should say which one this is, no? just leaving "integer" there after "uint64_t" is not helping much espcialy the class of clients you are thinking of.